src/org/labkey/targetedms/parser/skyd/ChromGroupHeaderInfo.java

```diff
@@ -45,52 +45,56 @@
     private double precursor;
     private long locationPoints;
     private int uncompressedSize;
-    private Float startTime;
-    private Float endTime;
+    private float startTime;
+    private float endTime;
     private float collisionalCrossSection;
 
     public ChromGroupHeaderInfo(CacheFormatVersion cacheFormatVersion, LittleEndianInput dataInputStream)
     {
+        // For reducing memory usage, following assignments to the above instance variables are commented out because of
```

Comment:
These variables need to be commented out above to actually reduce the memory footprint.